.github/workflows/gradle-wrapper-validation.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Validate Gradle Wrapper'
+on: [push, pull_request]
```

Comment:
I'm wondering if it would be enough to have only the `pull_request` trigger, wdyt?

Based on the [GitHub actions documentation](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push) for the `push` trigger:
> Runs your workflow when you push a commit or tag.

I understand that this trigger would only cover the cases of pushing a commit or tag on a branch without a pull request.

Reply:
We have been using `[push, pull_request]` as recommended by Gradle. The check only takes a few seconds, so I think it's worth being consistent, but I can change it if you'd like.

Reply:
Yep, I don't think having two PR checks would really impact dev workflow as it only takes around 10 seconds. Ok, let's keep with the recommendation from Gradle.